backend/settings.py

```diff
@@ -156,14 +151,12 @@
 
 # Celery config
 CELERY_RESULT_BACKEND = 'django-db'
-CELERY_BROKER_URL = 'redis://localhost:6379/0'
+CELERY_BROKER_URL = os.environ.get('CELERY_BROKER_URL', 'redis://localhost:6379/0')
```

Comment:
can you test something? What happens if we leave this as `CELERY_BROKER_URL = 'redis://localhost:6379/0'` and provide the env override anyways? Does the env override work without using `os.environ.get`?

Reply:
It does not. My docker compose workers were failing to connect until I changed this